packages/loaders/src/utils/animations.js

```diff
@@ -1,32 +1,97 @@
+/* stylelint-disable rule-empty-line-before */
```

Comment:
Prefer moving this directly adjacent to the code it is disabling.

Reply:
It's the whole file

Reply:
But doesn’t include the copyright header. Hopefully just above the keyframes